data/sheragi/archaeology missions.txt

```diff
@@ -204,8 +210,20 @@ mission "Deep Archaeology 5"
 		personality heroic
 		government Deep
 		ship "Raven (Afterburner)" "D.P.S. Wyrd"
+		to spawn
+			not "dps wyrd dead"
```

Comment:
I think it makes more sense to use `to despawn` than `to spawn`, because if this condition is true, it will then delete the entire NPC before the mission even starts, whereas when using `to spawn`, the game is expecting the NPC to be capable of spawning at some point during the mission, and therefore holds onto it.
```suggestion
		to despawn
			has "dps wyrd dead"
```